packages/wrangler/src/deployment-bundle/esbuild-plugins/hybrid-nodejs-compat.ts

```diff
@@ -193,16 +193,21 @@ function handleNodeJSGlobals(
  * This is important to support `inject` config for `performance` and `Performance` introduced
  * in https://github.com/unjs/unenv/pull/257
  */
-function encodeToLowerCase(str: string): string {
-	return str.replaceAll(/[A-Z]/g, (letter) => `$${letter.toLowerCase()}`);
+export function encodeToLowerCase(str: string): string {
+	return str
+		.replaceAll(/\$/g, ($) => "$$")
```

Comment:
```suggestion
		.replaceAll(/\$/g, () => "$$")
```

Reply:
Or even:

```suggestion
		.replaceAll(/\$/g, "$$")
```

Right?